### PR TITLE
Added user/get endpoint #52

### LINKED ISSUE #52: Add an endpoint to get the username of another user
**Is your feature request related to a problem? Please describe.**
We need to get away from UUIDs in the frontend. There is currently no way to display users with user names, we have to display UUIDs and that's not pretty and not at all informative.

**Describe the solution you'd like**
Add a new endpoint that takes a UUID and responds with the username, maybe the last-seen date and later possibly the online status of the corresponding user.

**Additional Context**
Please implement this endpoint inside the [server](https://github.com/cryptic-game/java-backend-dev/blob/master/server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java).

### PR DESCRIPTION
**Pull request by Cr7mbl3**
**Description**
Added a new endpoint (user/get_user) that takes a UUID and responds with the username and last-seen date of the corresponding user.

**Issue**
Closes #52 

**Additional Notes**
Maybe add the online status to the response.

### CODE REVIEW

File: server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java [hunk 1]
```diff
@@ -183,4 +183,23 @@ public ApiResponse delete(final ApiClient client, @ApiParameter("password") fina
 
         return new ApiResponse(ApiResponseType.OK);
     }
+
+    @ApiEndpoint("get_user")
```
Comment (JannikEmmerich) [inline server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java:187]: rename to `user/get`

File: server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java [hunk 2]
```diff
@@ -183,4 +183,23 @@ public ApiResponse delete(final ApiClient client, @ApiParameter("password") fina
...
+    public ApiResponse getUser(final ApiClient client, @ApiParameter("uuid") final UUID uuid) {
+        Session session = client.get(Session.class);
+        if (session == null || !session.isValid()) {
+            return new ApiResponse(ApiResponseType.FORBIDDEN, "NOT_LOGGED_IN");
+        }
+
+        session = Session.getById(uuid);
```
Comment (JannikEmmerich) [inline server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java:194]: It's about getting the serialized user information by its id not about getting a user by a session.

File: server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java [hunk 3]
```diff
@@ -183,4 +183,21 @@ public ApiResponse delete(final ApiClient client, @ApiParameter("password") fina
...
+            return new ApiResponse(ApiResponseType.FORBIDDEN, "NOT_LOGGED_IN");
+        }
+
+        final User user = User.getById(userId);
+
+        if (user == null) {
+            return new ApiResponse(ApiResponseType.NOT_FOUND, "USER_NOT_FOUND");
+        }
+
+        return new ApiResponse(ApiResponseType.OK, JsonBuilder.create("username", user.getName())
```
Comment (JannikEmmerich) [inline server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java:200]: Rename `username` to `name`, `last_seen` to `last` and add `id`.

Comment (MarcelCoding) [reply server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java:200]: Sezalisize the user by submitting the user to JsonBuilder without any other actions that replaces the idear of @JannikEmmerich

**Pull request merged**